app/extensions/git_store/__init__.py

```diff
@@ -182,6 +182,12 @@ def filter_for(cls, obj, git_stores):
     def ensure_remote_delay(cls, obj):
         raise NotImplementedError()
 
+    @classmethod
+    def get_elasticsearch_schema(cls):
+        from app.modules.assets.schemas import DetailedGitStoreSchema
```

Comment:
Probably not specific to this change but the fact that GitStore schemas are in the Asset Module needs more comments there. The reasoning is certainly not obvious to me.

Reply:
These have been moved to `app/extensions/git_store/schemas.py`.  The original reasoning was that the schema was not specific to an `AssetGroup` and was moved into the `Asset` module so that `Missions` could use it more natively.  The new `git_store` extension has the model definitions and now the schema definitions, which is more appropriate.